io.home-assistant.binary-sensor.garage-door/dataset.tt

```diff
@@ -0,0 +1,118 @@
+dataset @io.home-assistant.binary-sensor.garage-door language "en" {
```

Comment:
We shouldn't map this to a binary-sensor garage door. Instead, we should map this to the garage door.

Reply:
I'm not sure what you mean here. Are you suggesting we merge the "cover" and "binary sensor" Home Assistant entities into a single Thingpedia device?

Reply:
Well yes and no 😄 

Because Almond has no security controls yet, we shouldn't expose the garage doors/doors/locks controls in Almond yet. So yeah, we could merge them.

Reply:
I'm not opposed to merging them, but that runs somewhat contrary to the current design of "one entity = one Thingpedia device". Should we instead go by "one device class = one Thingpedia device"? Is there a registry of all device classes, regardless of entities?
And, eg. is it guaranteed that the cover entity for door also always comes with the binary sensor entity? Or can they come and go more or less independently?

Reply:
My proposal as outlined in our doc was indeed 1 device class -> 1 thingpedia device. The reason for that is that each device class requires different utterances to refer to it.

Reply:
Right, but I think what we implemented was 1 pair of (device class, entity) -> 1 thingpedia device. Which is probably suboptimal from a user's point of view...